tests/python/test_relational_vcfs.py

```diff
@@ -0,0 +1,90 @@
+import nose
+
+import tempfile
+
+
+
+def test_vcf_to_csv():
+    import vcf, csv
+    from common.relational_vcf import vcf_to_csv
+    vcfreader = vcf.Reader(open('tests/data/snv.vcf'))
+    columns = ['contig', 'position', 'sample:DP', 'info:VAF', 'info:DP', 'something']
+    filename = vcf_to_csv(vcfreader, columns, None,
+                          default_values={'something': 'Something!'})
+
+    with open(filename) as fd:
```

Comment:
That page doesn't use the word "descriptor" or `fd`. Wikipedia seems to agree that they're ints: [File descriptor](http://en.wikipedia.org/wiki/File_descriptor).

I'd be if the var was just called `f`. I'm bringing this up because I began reading this code assuming that `fd` was a number and got confused.